demonstrations/tutorial_noisy_circuits.py

```diff
@@ -250,7 +250,7 @@ def damping_circuit(x):
 
 
 def cost(x, target):
-    return (damping_circuit(x) - target)**2
+    return (damping_circuit(x) - qml.numpy.array(target, requires_grad=True))**2
```

Comment:
the second arg passed to `cost` below (`ev`) is explicitly marked as `requires_grad=False`. Should we reflect that change here?
```suggestion
    return (damping_circuit(x) - qml.numpy.array(target, requires_grad=False))**2
```
not sure why it doesn't come in with the right shape. The originally passed value is an `np.tensor`, any reason to change that?

Reply:
I changed everything to float and it works, autograd mysteries